### DISCUSSION TITLE
Send notifications to the users regarding mobile app

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#346] [General Discussion]
**Scenario**
My app is live on playstore, and there are 10 active users. I want to send a notification/message to the active users about the update/new feature of app.

Can anyone help me to understand how to send push notification? Or is there any configuration or in built function available to send notifications on mobile?

to: @swayangjit 
cc: @baljeetsingh

**Comment by Krishnaj20** [comment]
you can use firebase to trigger these notifications.

@ashokreddy1208 @anandvarada - can you share details about how DIKSHA is using firebase to send these notifications. 

@vinukumar-vs @swayangjit

**Comment by Krishnaj20** [comment]
@ashokreddy1208 @anandvarada - can you help here please?

**Reply by nidhikachhawa24** [reply]
Can anyone help me on this?

cc: @ashokreddy1208 @anandvarada @Krishnaj20

**Comment by ashokreddy1208** [comment]
@amitpriyadarshi  can you help here?
